src/gei/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -165,16 +165,16 @@ private string ExtractGhesBaseUrl(string ghesApiUrl)
       bool lockSourceRepo,
       bool blobCredentialsRequired)
     {
-        var gitDataArchiveId = await _sourceGithubApi.StartGitArchiveGeneration(githubSourceOrg, sourceRepo);
-        _log.LogInformation($"Archive generation of git data started with id: {gitDataArchiveId}");
+        var gitSourceMigrationsApiId = await _sourceGithubApi.StartGitArchiveGeneration(githubSourceOrg, sourceRepo);
```

Comment:
as I mentioned [here](https://github.com/github/gh-gei/pull/786#issuecomment-1407258606) using the term `ApiId` is misleading. I really think the variable name was good as is.